packages/google-tag-manager-analytics/src/components/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -16,19 +16,16 @@ exports[`GoogleTagManager works with a single container id 1`] = `
 
 exports[`GoogleTagManager works with a single container id 2`] = `
 <noscript>
-  <iframe
-    css={
+  <style
+    dangerouslySetInnerHTML={
       Object {
-        "map": undefined,
-        "name": "8s6lqv",
-        "next": undefined,
-        "styles": "
-          display: none;
-          visibility: hidden;
-        ",
-        "toString": [Function],
+        "__html": ".css-llz5b9-GtmCode{display:none;visibility:hidden;}",
       }
     }
+    data-emotion="css llz5b9-GtmCode"
+  />
+  <iframe
+    className="css-llz5b9-GtmCode"
```

Comment:
Any idea about why this component has a label in the `className`, even though the `autoLabel` option of `@emotion/babel-plugin` is set to `never` in the `jest.babel.config.json` file?

Just curious. :thinking:

Reply:
Thank you @DAreRodz 🙏  for the review.

> Any idea about why this component has a label in the className, even though the autoLabel option of @emotion/babel-plugin is set to never in the jest.babel.config.json file?

It's because of this issue: https://github.com/emotion-js/emotion/pull/2157

> In that case, I guess we would have to remove the "babel-plugin-frontity" from the other examples as well, right? Not only from the mars theme example.

Honestly I was about to, but then was talking with @luisherranz and if I remember correctly the examples will be moved or maintained by the devrel team. I could easily remove them now. Should I?

Reply:
> It's because of this issue: [emotion-js/emotion#2157](https://github.com/emotion-js/emotion/pull/2157)

Good to know. Thanks! :raised_hands: 

> I could easily remove them now. Should I?

Yes, please. It's very simple to fix and that way we don't have to remember to notify the devrel team about this.

Reply:
Done ✅